x-pack/plugins/discover_log_explorer/public/deep_links.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AppNavLinkStatus, DEFAULT_APP_CATEGORIES } from '@kbn/core/public';
+import { i18n } from '@kbn/i18n';
+import { LOG_EXPLORER_PROFILE_ID } from '../common/constants';
+
+export const getLogExplorerDeepLink = ({ isVisible }: { isVisible: boolean }) => ({
+  id: LOG_EXPLORER_PROFILE_ID,
```

Comment:
Is this the right id to use as the deep link id?

Reply:
I'd say so because it represents the path associated with the registered profile and the profile id used by the profile registry. Also, when building the deepLinks `link` used to match them in the navigation tree, this is concatenated with the parent app id by a `:`, finally resulting in `discover:log-explorer`.